calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -79,21 +83,29 @@ public void ShowNumberOnDisplay(string textToDisplay)
         public void ClearScreen()
         {
             DisplayContent = "0";
+            _hasDecimal = false;
         }
 
-        public void ShowPlusOnDisplay(string plus)
+        private void AssignOperator(string newOperator)
         {
+            _number1 = Convert.ToDouble(DisplayContent);
+            this._operator = newOperator;
+            ClearScreen();
+        }
 
+        public void ShowPlusOnDisplay(string plus)
+        {
+            AssignOperator(plus);
         }
 
         public void ShowMultiplyOnDisplay(string multiply)
         {
-
+            AssignOperator(multiply);
```

Comment:
What happened to "Stringly-Typed" vs "Strongly-Typed?"

Reply:
I want to add... if you're committed to StringlyTyped, I highly recommend you create a CommandParameter for each of the operators and consolidate the operator commands to a single command.